spec/helpers/admin/translated_record_helper_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe Admin::TranslatedRecordHelper, type: :helper do
+  describe '#translated_form_for' do
+    subject do
+      helper.translated_form_for(record) { |f| '' }
+    end
+
+    let(:record) { FactoryBot.create(:announcement) }
+
+    let(:expected) do
+      helper.form_for(record, builder: Admin::TranslatedRecordForm) { |f| '' }
+    end
+
+    it { is_expected.to eq(expected) }
```

Comment:
This spec isn't proving anything without the new form builder methods being used. We don't want to expect those methods are outputting the HTML here so I would consider using a `to receive` or `to have_received` matcher to check the `builder` argument is used.